_posts/2023-10-20-anvio-org-tutorial.md

```diff
@@ -0,0 +1,168 @@
+
+{:.notice}
+If you already wrote a blog post on a different website and you want to add a link to it on anvio.org, you don't have to go through this whole tutorial. The only section that is really relevant is [the one about adding your post to the resources list](#step-5-include-the-post-in-a-yaml-file-under-_dataresources). You should follow that section to include the external URL to your existing post in the appropriate file within the repository, then send us a pull request.
+
+## Step 1) Get the anvio.org GitHub repository on your computer
+
+To add your content, you'll need to have a local copy of the materials for this website on your computer. The website is hosted on Github in [this repository](https://github.com/merenlab/anvio.org). You should download this repository.
+
+Once you have it, please follow the installation commands for Ruby that are described [here](https://anvio.org/docs/anvio-docs-tutorial/#a-bit-of-setup). Once you are able to run the following command to run a local copy of the website, 
+
+```bash
+# run this command while inside the anvio.org directory
+bundle exec jekyll serve --incremental
+```
+
+you should be good to go and can come back to this page.
+
+## Step 2) Make a branch for adding your post
+
+It is best practice to write your post within a branch of the anvio.org repository and then submit a pull request (PR) to add it once it is all done. Make sure to get the latest changes to the website by running `git pull` first. Here is an example set of commands to start a branch for your post:
+
+```bash
+cd anvio.org
+git pull
+git checkout -b anvio-org-tutorial
+```
+
+Now you can start adding your contributions. From this point forward, we'll assume that your working directory in the terminal is the `anvio.org/` directory and that you are on the branch you created.
+
+## Step 3) Some helpful resources
+
+You may want to check out the following webpages if you aren't familiar with writing in Markdown or how Jekyll websites work:
+
+- [Markdown Guide](https://www.markdownguide.org/)
+- [Jekyll Documentation](https://jekyllrb.com/docs/) (helpful for understanding the website structure and commands for building it)
+
+Our website also includes several fancy syntax features that you can use to do things like making 'additional info' boxes, linking to the anvi'o documentation, and adding quotes or images. We highly recommend that you always keep our 'Web Tips' page open in a tab of your browser so that you can copy-paste the syntax for adding these features:
+
+- [Tips for web developers](https://merenlab.org/web-tips/)
```

Comment:
I think you mean https://anvio.org/web-tips/ ? :)

Reply:
hehe, I didn't even realize we had an equivalent page for that on anvio.org. I switched the links :)